Sources/CADCacheAdvance/CADCacheAdvance.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  Created by Dan Federman on 4/24/20.
+//  Copyright © 2020 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#if SWIFT_PACKAGE_MANAGER
+import CacheAdvance
```

Comment:
Why does the generated Xcode project not need this?

Reply:
The generated project actually does get this import. `SWIFT_PACKAGE_MANAGER` is defined in `Package.swift`:
https://github.com/dfed/CacheAdvance/blob/345ca001533be5599aa2e000ea4a9134fbb0b1da/Package.swift#L28

This setting applies to any Xcode project that is derived from the `Package.swift`, i.e. an Xcode project created by double-clicking the `Package.swift`, or a generated package created via `swift package generate-xcodeproj`. Basically any Xcode project created by Swift Package Manager gets this flag.

However, our Cocoapods release doesn't need this flag. Cocoapods ships a single library, rather than a separate library for `CADCacheAdvance`, `CacheAdvance`, and `SwiftTryCatch`. Therefore, we can't include `import` statements between our SPM libraries in our Cocoapods package or we'd introduce a compiler error.

Reply:
I wonder if CocoaPods subspecs would avoid the need for this?

Reply:
A code comment explaining this would be helpful.

Reply:
My understanding (which could be wrong) is that subspecs conditionally include code, but don't make new modules/libraries/targets. As such, I don't think subspecs would help. But def worth adding a comment. Will do.

Reply:
I think you're probably right re: subspecs